### ISSUE TITLE
Bug: onVerify callback attempts to run when undefined

### ISSUE THREAD

**Issue by RamennBowls** [hCaptcha/react-hcaptcha#169]
It appears when executing the captcha asynchronously, the `handleSubmit` function (in [src/index.js:226](https://github.com/hCaptcha/react-hcaptcha/blob/master/src/index.js#L226)) doesn't check whether or not the property `onVerify` is set resulting with this error:
```
[hCaptcha] There was an error in your callback.
TypeError: onVerify is not a function
```

<details>
<summary><b><a href="https://github.com/hCaptcha/react-hcaptcha/blob/master/src/index.js#L226">src/index.js:226</a></b></summary>

https://github.com/hCaptcha/react-hcaptcha/blob/701c1bf5b1316e3f2bf9e24606f30ff597f745b0/src/index.js#L222-L231

</details>

---
There is a temporary solution (for anyone who might need it), just use an empty arrow function like so (>ᴗ•)b
```ts
<HCaptcha onVerify={() => {}} />
```

**Comment by zoryana94** [comment]
Hi @RamennBowls !

Thank you for notifying us about the issue!
We'll look into it and make sure to keep you posted.

Best Regards,
IMI Dev Team

**Comment by zoryana94** [comment]
Hi @RamennBowls,

To keep you posted, the team will fix it due to the priorities.
I should also let you know that we really appreciate the contributions to the project. So please feel free to submit the PR with a fix.

Best Regards,
IMI Dev Team

**Comment by zoryana94** [comment]
Hi @RamennBowls ,

The issue should be fixed in [our latest 1.6.1 release.](https://github.com/hCaptcha/react-hcaptcha/releases/tag/1.6.1)
Please test it from your side and let us know if it works for you.

Thank you,
IMI Dev Team
